### PR TITLE
vaos systems endpoint

### PR DESCRIPTION
**Pull request by kreek**
## Description of change
Adds endpoint and service to retrieve VAOS system identifiers for a user (on the VAOS side they use the JWT token to look up the user). Also adds skeleton for swagger docs for VAOS and the systems request and schema block files.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/1782

## Testing done
unit tests

## Testing planned
staging integration tests

#### Unique to this PR
- [x] Adds GET `/service/vaos/v0/systems`
- [x] Adds VAOS Systems Service
- [x] Swagger docs for VAOS

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -181,4 +181,6 @@ def jobs
   config.after(:all) do
     FileUtils.rm_rf(Dir[Rails.root.join('spec', 'support', 'uploads')]) if Rails.env.test?
   end
+
+  config.fuubar_auto_refresh = false
```
Comment (kreek) [inline spec/rails_helper.rb]: makes `binding.pry` work again

File: modules/vaos/config/routes.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 VAOS::Engine.routes.draw do
   namespace :v0, defaults: { format: 'json' } do
-    get 'appointments', to: 'vaos#get_appointments'
```
Comment (kreek) [inline modules/vaos/config/routes.rb]: This snuck in earlier even though there was no appointments endpoint yet. Another PR will soon add it back in, so... 🤷‍♂

File: modules/vaos/app/controllers/vaos/v0/vaos_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module VAOS
+  module V0
+    class VAOSController < ApplicationController
+      before_action { authorize :vaos, :access? }
+
+      def get_systems
```
Comment (saneshark) [inline modules/vaos/app/controllers/vaos/v0/vaos_controller.rb]: should we maybe just stick to restful resources? 

```
class SystemsController < ApplicationController
      before_action { authorize :vaos, :access? }

      def index
      ...
```

File: modules/vaos/app/controllers/vaos/v0/vaos_controller.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        response = systems_service.get_systems(current_user)
+        render json: VAOS::SystemSerializer.new(response)
```
Comment (saneshark) [inline modules/vaos/app/controllers/vaos/v0/vaos_controller.rb]: collections should have a options[:meta], no?

File: spec/rails_helper.rb (hunk 1 above)
Comment (annaswims) [reply spec/rails_helper.rb]: There's another PR that might help this too https://github.com/department-of-veterans-affairs/vets-api/pull/3496

File: modules/vaos/app/policies/vaos_policy.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+VAOSPolicy = Struct.new(:user, :vaos) do
+  def access?
+    Flipper.enabled?('va_online_scheduling', user) && user.loa3?
```
Comment (annaswims) [inline modules/vaos/app/policies/vaos_policy.rb]: Please consider `va_online_scheduling` to `config/features.yml` with a description.  While what you have is practically the same thing, I'd prefer to see all disabled features in the admin UI.

**Pull request merged**